modules/react/collection/lib/useListItemAllowChildStrings.ts

```diff
@@ -0,0 +1,37 @@
+import {createElemPropsHook} from '@workday/canvas-kit-react/common';
+import {useListModel} from '@workday/canvas-kit-react/collection';
+
+/**
+ * This elemProps hook allows for children values to be considered identifiers if the children
+ * are strings. This can be useful for autocomplete or select components that allow string values.
+ * This hook must be defined _before_ {@link useListItemRegister} because it sets the `data-id`
```

Comment:
Small thing: "because it sets" -> "it" is ambiguous, do we mean "this hook" or "that hook"? Might be best to state either "this hook" or `useListItemRegister` to be clear.

Reply:
Also, we say "defined before" but it seems to be the opposite in the `composeHooks` usage. I understand that what comes later takes precedence (or do I?). I think maybe we mean something like "listed after" and maybe have added explanation that it will be applied in reverse order with `composeHooks`.
```